engine/src/player-platform.cpp

```diff
@@ -1876,6 +1876,9 @@ void MCPlayer::setenabledtracks(uindex_t p_count, uint32_t *p_tracks_id)
 				t_enabled = false;
 				MCPlatformSetPlayerTrackProperty(m_platform_player, i, kMCPlatformPlayerTrackPropertyEnabled, kMCPlatformPropertyTypeBool, &t_enabled);
 			}
+            
+            if (p_tracks_id == nil)
+                return;
```

Comment:
Hi @livecodepanos I think this fix is incorrect. The loop should not access array elements beyond the end of p_tracks_id, however because it is bound by t_track_count rather than p_count it will do that if the number of tracks is greater than the number of items in enabledTracks